sdk/src/main/java/com/akkaserverless/javasdk/AkkaServerless.java

```diff
@@ -454,6 +451,20 @@ public AkkaServerless registerValueEntity(
         entityClass, descriptor, ValueEntityOptions.defaults(), additionalDescriptors);
   }
 
+  public <T> AkkaServerless register(ValueEntityProvider<T> factory) {
+    return register(factory, ValueEntityOptions.defaults());
+  }
+
+  public <T> AkkaServerless register(ValueEntityProvider<T> factory, ValueEntityOptions options) {
```

Comment:
An alternative would be that we make ValueEntityProvider a builder, so that options can be specified via the ValueEntityProvider. 

```
ValueEntityProvider.of(context -> new CustomerValueEntity()).withOptions(...)
```

The advantage is that it's easier for us to evolve the builder with more optional properties, than having overloaded register methods.